kubernetes/provider.go

```diff
@@ -299,8 +299,9 @@ func Provider() *schema.Provider {
 			"kubernetes_csi_driver_v1":    resourceKubernetesCSIDriverV1(),
 
 			// provider helper resources
-			"kubernetes_labels":      resourceKubernetesLabels(),
-			"kubernetes_annotations": resourceKubernetesAnnotations(),
+			"kubernetes_labels":             resourceKubernetesLabels(),
+			"kubernetes_annotations":        resourceKubernetesAnnotations(),
+			"kubernetes_config_map_v1_data": resourceKubernetesConfigMapV1Data(),
```

Comment:
Most of our resources have a version at the end of their names, shall we continue this practice and call it `kubernetes_config_map_data_v1`? I am not sure though if that one won't overlap with any native Kubernetes resources in the future. I am totally fine with the current name and can see the logic here too, so I am just curious.